test.py

```diff
@@ -0,0 +1,30 @@
+from GeneratorPgn import IteratorPgn, Preprocessor, ArrayToFen
+import chess
+import numpy as np
+"""
+tests whether the downloaded fen record after processing into a list and then restored again has not been deformed, and whether the given fen record may mean a real position
+"""
+number_of_tests=1000
+test_number=0

```

Comment:
please format this file using black, it has some formatting mistakes (no spaces around operators etc)